src/identity/components/internal/WithNameBadge.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Address } from 'viem';
+import { useVerified } from '../../hooks/useVerified';
+import { Badge } from './Badge';
+
+type DisplayBadgeProps = {
+  children: React.ReactNode;
+  address: Address;
+};
+
+function DisplayBadge({ children, address }: DisplayBadgeProps) {
```

Comment:
We are going to have multiple badges, so if a badge is assigned to a schema, than we should probably define a schema in `DisplayBadge`